lib/template.js

```diff
@@ -1,14 +1,16 @@
 var template = require('lodash.template');
-var reInterpolate = require('lodash._reinterpolate');
+var reEscape = require('lodash.reescape');
```

Comment:
Your code shouldn't break it just momentarily has an extra dep for each of the proxies regexp deps.